lib/node_modules/@stdlib/number/float64/base/ulp-difference/examples/index.js

```diff
@@ -33,3 +33,6 @@ console.log( d );
 
 d = ulpdiff( 0.0, -0.0 );
 console.log( d );
+
```

Comment:
@anandkaranubc Mind also adding return annotations in this file, as well? This adds another form of "unit testing".